activities/communication/community-discussions.md

```diff
@@ -0,0 +1,23 @@
+---
+type: Resource
+---
+
+# Community discussions
+
+We have a forum for community discussions at: [discuss.publiccode.net](https://discuss.publiccode.net).
+
+## Purpose
+
+The forum is a space for our community to talk publicly about topics that does not naturally fit anywhere else.
+
+## When to not use the forum
+
+The forum doesn't replace:
+
+- making issues on Github
+- our internal chat (Odoo Discuss)
```

Comment:
I would remove the name of the tool here. Those who need to know will understand and it doesn't turn into technical debt when we decide to change tools. Also I'd use 'staff' and not 'internal' since it is a clearer qualitfier.

```suggestion
- our staff chat
```